*** ./smali/android/content/pm/PackageParser.smali	2017-03-30 17:56:14.817796343 +0530
--- /home/nitesh/patchrom/tomato/temp/new_smali/framework.jar.out/./smali/android/content/pm/PackageParser.smali	2017-03-30 17:57:59.997797266 +0530
***************
*** 9660,9666 ****
  
      .local v28, "parser":Landroid/content/res/XmlResourceParser;
      :try_start_0
!     new-instance v31, Landroid/content/res/Resources;
  
      move-object/from16 v0, p0
  
--- 9660,9666 ----
  
      .local v28, "parser":Landroid/content/res/XmlResourceParser;
      :try_start_0
!     new-instance v31, Landroid/content/res/MiuiResources;
  
      move-object/from16 v0, p0
  
***************
*** 9672,9678 ****
  
      move-object/from16 v1, p2
  
!     invoke-direct {v0, v1, v5, v6}, Landroid/content/res/Resources;-><init>(Landroid/content/res/AssetManager;Landroid/util/DisplayMetrics;Landroid/content/res/Configuration;)V
      :try_end_0
      .catch Landroid/content/pm/PackageParser$PackageParserException; {:try_start_0 .. :try_end_0} :catch_3
      .catch Ljava/lang/Exception; {:try_start_0 .. :try_end_0} :catch_1
--- 9672,9678 ----
  
      move-object/from16 v1, p2
  
!     invoke-direct {v0, v1, v5, v6}, Landroid/content/res/MiuiResources;-><init>(Landroid/content/res/AssetManager;Landroid/util/DisplayMetrics;Landroid/content/res/Configuration;)V
      :try_end_0
      .catch Landroid/content/pm/PackageParser$PackageParserException; {:try_start_0 .. :try_end_0} :catch_3
      .catch Ljava/lang/Exception; {:try_start_0 .. :try_end_0} :catch_1
