*** ./smali/com/android/server/am/ActivityManagerService.smali	2017-03-30 17:55:50.881796132 +0530
--- /home/nitesh/patchrom/tomato/temp/new_smali/services.jar.out/./smali/com/android/server/am/ActivityManagerService.smali	2017-03-30 17:57:35.789797054 +0530
***************
*** 82979,82984 ****
--- 82979,82990 ----
  
      invoke-direct/range {v3 .. v19}, Lcom/android/server/am/ActivityManagerService;->broadcastIntentLocked(Lcom/android/server/am/ProcessRecord;Ljava/lang/String;Landroid/content/Intent;Ljava/lang/String;Landroid/content/IIntentReceiver;ILjava/lang/String;Landroid/os/Bundle;[Ljava/lang/String;ILandroid/os/Bundle;ZZIII)I
  
+     move/from16 v0, v22
+ 
+     move-object/from16 v1, v29
+ 
+     invoke-static {v0, v1}, Lcom/android/server/am/ActivityManagerServiceInjector;->handleExtraConfigurationChangesForSystem(ILandroid/content/res/Configuration;)V
+ 
      and-int/lit8 v3, v22, 0x4
  
      if-eqz v3, :cond_7
